### ISSUE TITLE
cannot using as a function in browser: Uncaught TypeError: Class constructors cannot be invoked without 'new'

### ISSUE THREAD

**Issue by richex-cn** [darkXmo/once-init#5]
```html
<script src="once-init@1.2.1/dist/index.js"></script>
<script>
  const get = oi(...) // Uncaught TypeError: Class constructors cannot be invoked without 'new'
</script>
```

Maybe can be mount a helper function to `OnceInit` namespace? Like `OnceInit.wrap(...)`.

**Comment by darkXmo** [comment]
Have solved by commit `441f053ad4993588c9d3db032e257d7e077cd6fa` and `f8f2242663614161497d5559d4520bf77a416fb7`

**Comment by richex-cn** [comment]
Nice~

441f053ad4993588c9d3db032e257d7e077cd6fa
f8f2242663614161497d5559d4520bf77a416fb7
